### PR TITLE
Bump to 2.24.0, based on Fluent Bit 1.9.3

### PR DESCRIPTION
**Pull request by matthewfala**
Signed-off-by: Matthew Fala <falamatt@amazon.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+* Fluent Bit [1.9.3](https://fluentbit.io/announcements/v1.9.3/)
+* Amazon CloudWatch Logs for Fluent Bit 1.7.0
+* Amazon Kinesis Streams for Fluent Bit 1.9.0
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Compared to `2.23.4` this release adds:
+* Bug - Resolve IMDSv1 fallback error introduced in 2.21.0 [aws-for-fluent-bit:259](https://github.com/aws/aws-for-fluent-bit/issues/259)
+* Bug - Cloudwatch Fix integer overflow on 32 bit systems when converting tv_sec to millis [fluentbit:3640](https://github.com/fluent/fluent-bit/issues/3640)
+* Enhancement - Only create Cloudwatch Logs log group if it does not already exist to prevent throttling [fluentbit:4826](https://github.com/fluent/fluent-bit/pull/4826)
+* Enhancement - Implement multiline buffered mode for docker log driver partial message support [fluentbit:4671](https://github.com/fluent/fluent-bit/pull/4671)
```
Comment (PettitWesley) [inline CHANGELOG.md:15]: the buffered mode already existed in 1.8 series, just say implemented docker log driver partial message support

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 FROM public.ecr.aws/amazonlinux/amazonlinux:latest as builder
 
 # Fluent Bit version; update these for each release
-ENV FLB_VERSION 1.8.15
+ENV FLB_VERSION 1.9.3
 # branch to pull parsers from in github.com/fluent/fluent-bit-docker-image
 ENV FLB_DOCKER_BRANCH 1.8
```
Comment (zhonghui12) [inline Dockerfile:6]: we should do 1.9 here?

File: Dockerfile.debug [hunk 3]
```diff
@@ -1,7 +1,7 @@
 FROM public.ecr.aws/amazonlinux/amazonlinux:latest as builder
 
 # Fluent Bit version; update these for each release
-ENV FLB_VERSION 1.8.15
+ENV FLB_VERSION 1.9.3
 # branch to pull parsers from in github.com/fluent/fluent-bit-docker-image
 ENV FLB_DOCKER_BRANCH 1.8
```
Comment (zhonghui12) [inline Dockerfile.debug:6]: +1 see above

**Review by zhonghui12 — COMMENTED** [review]
Nit: for commit msg, I guess "Bump to 2.24.0" is enough? Just try to keep consistent

File: Dockerfile (hunk 2 above)
Comment (matthewfala) [reply Dockerfile:6]: O i c. Good catch

File: CHANGELOG.md (hunk 1 above)
Comment (matthewfala) [reply CHANGELOG.md]: OK, done.

Not mentioning the priority event loop, since it's not aws specific.

**Review by PettitWesley — APPROVED** [review]

**Review by zhonghui12 — APPROVED** [review]

**Pull request merged**